.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# See Dependabot documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/dev/ci"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "jmagman"
```

Comment:
@godofredoc Should someone from infra be added as a backup?

Reply:
sounds good, please add @keyonghan and @yusufm